README.md

```diff
@@ -752,6 +752,7 @@ Table of Contents
 
 ## STUN, WebRTC, Web Socket Servers and Other Routers
 
+   * [TryCloudflare](https://developers.cloudflare.com/argo-tunnel/trycloudflare) — Expose local HTTP servers through Argo Tunnel to public.
```

Comment:
I'd prefer we just update the existing one with additional information

Reply:
But the existing CloudFlare entries are about CDN, DNS and Workers, which is quiet different from this feature, or you want to add CloudFlare into major cloud and group them together?